Makefile

```diff
@@ -0,0 +1,13 @@
+SHELL := bash -euo pipefail
+
+.PHONY: local-image
+
+local-image:
+	./devel/start-localhost-registry || true
+	./devel/build
```

Comment:
This uses [default argument values](https://github.com/nextstrain/docker-base/blob/1b0d1998c8cb03cbd0b5eb630fcd86fdfb247c99/devel/build#L16-L19), which includes `platform=linux/amd64`. I develop on Apple silicon and want to use `platform=linux/arm64` for faster builds. This `make` shortcut doesn't provide a way to specify that.

Maybe the default platform in `devel/build` can be inferred from the host machine? That way, this `make` shortcut would build without emulation on both `amd64` and `arm64`.

Reply:
Ahh, yeah, that's a good point. I'll make adjustments.

Reply:
@victorlin see https://github.com/nextstrain/docker-base/pull/159/commits/c2921fe32302583b728b5f592a11112a88568db8